web/src/components/IntroScreen/IntroScreen.tsx

```diff
@@ -1,50 +1,16 @@
-import React, { useState, useEffect, useRef } from 'react'
-import { Redirect, useHistory } from 'react-router-dom'
-import { AppClient } from '@holochain/client'
+import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
 
 import Button from '../Button/Button'
 import Icon from '../Icon/Icon'
 import Typography from '../Typography/Typography'
-import { Profile } from '../../types'
-import { WireRecord } from '../../api/hdkCrud'
-import { CellIdString } from '../../types/shared'
 
 import './IntroScreen.scss'
 
-export type IntroScreenStateProps = {
-  whoami: WireRecord<Profile>
-  profilesCellId: CellIdString
-}
-export type IntroScreenDispatchProps = {
-  fetchWhoami: (profilesCellId: CellIdString) => Promise<void>
-}
-export type IntroScreenOwnProps = {
-  appWebsocket?: AppClient
-}
-export type IntroScreenProps = IntroScreenOwnProps &
-  IntroScreenStateProps &
-  IntroScreenDispatchProps
 
-const IntroScreen: React.FC<IntroScreenProps> = ({
-  fetchWhoami,
-  whoami,
-  profilesCellId,
-}) => {
+const IntroScreen: React.FC<{}> = ({}) => {
   const history = useHistory()
 
-  /*
-    We do this so that if/when the agents Profile gossips to them,
-    having been already imported by someone else,
-    they don't stay here accidentally
-  */
-  const instance = useRef<NodeJS.Timeout>()
```

Comment:
We now don't have a global profiles dna anymore so there won't be anything gossiped to us if we start up a completely fresh instance of acorn.